Dockerfile

```diff
@@ -76,7 +76,7 @@ ENV PATH="$BUNDLE_BIN:$PATH:$BUNDLE_PATH/bin"
 RUN apt-add-repository ppa:brightbox/ruby-ng \
   && apt-get update \
   && apt-get install -y --no-install-recommends ruby2.7 ruby2.7-dev \
-  && gem update --system 3.3.11 --no-document \
```

Comment:
Per https://github.com/dependabot/dependabot-core/pull/5035#issue-1213736168, the `--no-document` flag is faster, so we should try adding that back in even with the held-back version.

I'll open a PR.